seacrowd/sea_datasets/indonesian_madurese_bible_translation/indonesian_madurese_bible_translation.py

```diff
@@ -0,0 +1,238 @@
+# However, if you need to access different files for each config you can have multiple entries in this dict.
+# This can be an arbitrarily nested dict/list of URLs (see below in `_split_generators` method)
+_URLS = {
+    _DATASETNAME: "https://prod-dcd-datasets-cache-zipfiles.s3.eu-west-1.amazonaws.com/cgtg4bhrtf-3.zip",
+}
+
+# TODO: add supported task by dataset. One dataset may support multiple tasks
+_SUPPORTED_TASKS = [Tasks.MACHINE_TRANSLATION]  # example: [Tasks.TRANSLITERATION, Tasks.NAMED_ENTITY_RECOGNITION, Tasks.RELATION_EXTRACTION]
+
+# TODO: set this to a version that is associated with the dataset. if none exists use "1.0.0"
+#  This version doesn't have to be consistent with semantic versioning. Anything that is
+#  provided by the original dataset as a version goes.
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+# TODO: Name the dataset class to match the script name using CamelCase instead of snake_case
+class NewDataset(datasets.GeneratorBasedBuilder):
+    """TODO: This corpus consists of more than 20,000 Indonesian - Madurese sentences."""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    # You will be able to load the "source" or "seacrowd" configurations with
+    # ds_source = datasets.load_dataset('my_dataset', name='source')
+    # ds_seacrowd = datasets.load_dataset('my_dataset', name='seacrowd')
+
+    # For local datasets you can make use of the `data_dir` and `data_files` kwargs
+    # https://huggingface.co/docs/datasets/add_dataset.html#downloading-data-files-and-organizing-splits
+    # ds_source = datasets.load_dataset('my_dataset', name='source', data_dir="/path/to/data/files")
+    # ds_seacrowd = datasets.load_dataset('my_dataset', name='seacrowd', data_dir="/path/to/data/files")
+
+    # TODO: For each dataset, implement Config for Source and SEACrowd;
+    #  If dataset contains more than one subset (see seacrowd/sea_datasets/smsa.py) implement for EACH of them.
+    #  Each of them should contain:
+    #   - name: should be unique for each dataset config eg. smsa_(source|seacrowd)_[seacrowd_schema_name]
+    #   - version: option = (SOURCE_VERSION|SEACROWD_VERSION)
+    #   - description: one line description for the dataset
+    #   - schema: options = (source|seacrowd_[seacrowd_schema_name])
+    #   - subset_id: subset id is the canonical name for the dataset (eg. smsa)
+    #  where [seacrowd_schema_name] can be checked in seacrowd/utils/constants.py
+    #    under variable `TASK_TO_SCHEMA`, in accordance to values from `_SUPPORTED_TASKS`
+    #    for all config(s) defined
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name="indonesian_madurese_bible_translation_source",
+            version=SOURCE_VERSION,
+            description="indonesian_madurese_bible_translation source schema",
+            schema="source",
+            subset_id="indonesian_madurese_bible_translation",
```

Comment:
I suggest using `_DATASETNAME` for `indonesian_madurese_bible_translation`

Reply:
done